config/log4j2-file.yaml

```
@@ -0,0 +1,21 @@
+Configuration:
+  status: WARN
+  Appenders:
+    RollingFile:
+      name: defaultFile
+      fileName: ${ksql.log.dir}/ksql-cli/cli-%timestamp.log
+      filePattern: ${ksql.log.dir}/ksql-cli/cli-%d{yyyy-MM-dd}-%i.log.gz
+      PatternLayout:
+        pattern: "[%d] %p %m (%c:%L)%n"
+      Policies:
+        TimeBasedTriggeringPolicy:
+          interval: 1
+        SizeBasedTriggeringPolicy:
+          size: 10MB
+      DefaultRolloverStrategy:
+        max: 5
```

nit: add comment for readability. 
```suggestion
        max: 5 # other files will be cleaned up.
```